Blockzilla/SiriShortcuts.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Intents
+import IntentsUI
+
+class SiriShortcuts {
+    enum activityType: String {
+        case erase
+        case eraseAndOpen = "org.mozilla.ios.Klar.eraseAndOpen"
+        case openURL
+    }
+    
+    func getActivity(for type: activityType) -> NSUserActivity? {
+        switch type {
+        case .eraseAndOpen:
+            return eraseAndOpenActivity
+        default:
+            break
+        }
+        return nil
+    }
+    
+    private var eraseAndOpenActivity: NSUserActivity? {
```

Comment:
nit: computed var vs. closure-initialized var, 
`eraseAndOpenActivity = { ... }()` seems clearer, a computed value indicates variability, and I after looking closer, I didn't see any variability.